File: frontend/src/app/entities/view/security.statistics.return.result.ts
export interface SecurityStatisticsReturnResult {
   annualisedPerformance: AnnualisedSecurityPerformance;
  summaryResult: SecurityStatisticsSummaryResult ;

}

export interface  AnnualisedSecurityPerformance {
  securityCurrency: string;
  mainCurrency: string;
  dateFrom: Date;
  dateTo: Date;
  lastYears: LastYears[];
  annualisedYears: AnnualisedYears[];
}

export interface SecurityStatisticsSummaryResult {
  dailyStandardDeviation: number;
}

export interface LastYears {
  year: number;
  performanceYear: number;
  performanceYearMC: number;
}

export interface AnnualisedYears {
  numberOfYears: number;
  performanceAnnualised: number;
  performanceAnnualisedMC: number;
}

File: backend/grafioschtrader-common/src/main/java/grafioschtrader/dto/SecurityStatisticsReturnResult.java
package grafioschtrader.dto;

public class SecurityStatisticsReturnResult {
  public AnnualisedSecurityPerformance annualisedPerformance;
  public SecurityStatisticsSummaryResult summaryResult;

  public SecurityStatisticsReturnResult(AnnualisedSecurityPerformance annualisedPerformance,
      SecurityStatisticsSummaryResult summaryResult) {
    this.annualisedPerformance = annualisedPerformance;
    this.summaryResult = summaryResult;
  }
   
}
